.github/workflows/ci.yml

```
@@ -49,6 +49,14 @@ jobs:
       # use the more efficient nextest
       - uses: taiki-e/install-action@nextest
       - uses: Swatinem/rust-cache@v2
+      - name: Install Circom
+        run: |
+          git clone https://github.com/iden3/circom.git
+          cd circom
+          cargo build --release
+          cargo install --path circom
+      - name: Execute compile.sh
+        run: bash ./src/frontend/circom/test_folder/compile.sh
```

This will take too many minutes each time, since it downloads and compiles from source code.
Since some months ago, we can directly download the compiled binaries from the releases.
Maybe we can change these lines by:

```suggestion
      - name: Download Circom
        run: |
          mkdir -p $HOME/bin
          curl -sSfL https://github.com/iden3/circom/releases/download/v2.1.6/circom-linux-amd64 -o $HOME/bin/circom
          chmod +x $HOME/bin/circom
          echo "$HOME/bin" >> $GITHUB_PATH
```

---

Thank you for telling me. It must be better.